### PR TITLE
Declaration of Page::sluggable() must be compatible with Sluggable::sluggable(): array

### PR DESCRIPTION
**Pull request by ccsliinc**
This declaration was no longer compatible with Cviebrock\EloquentSluggable\Sluggable::sluggable().  Need to type hint the function.

### CODE REVIEW

File: src/app/Models/Page.php [hunk 1]
```diff
@@ -37,7 +37,7 @@ class Page extends Model
      *
      * @return array
      */
-    public function sluggable()
+    public function sluggable():array
```
Comment (promatik) [inline src/app/Models/Page.php:40]:
```suggestion
    public function sluggable(): array
```

**Review by promatik — APPROVED** [review]
I can confirm [Sluggable method is type hinted](https://github.com/cviebrock/eloquent-sluggable/blob/master/src/Sluggable.php#L82);
`(): array` is PHP 7.0+, so I think it's safe to add this to Permission Manager

**Comment by promatik** [comment]
Thank you for this update @ccsliinc 🙌

**Comment by pxpm** [comment]
As reported here: https://github.com/Laravel-Backpack/CRUD/issues/3359

Php 8 now enforces that extending functions have the same `return type` defined.

**Pull request merged**